### PR TITLE
Update Kossel XL configuration for RC5

### PR DESCRIPTION
**Pull request by oxivanisher**
The example configuration for the Kossel XL is now compatible with 1.1 RC5.

### CODE REVIEW

File: Marlin/example_configurations/delta/kossel_xl/Configuration.h [hunk 1]
```diff
@@ -350,15 +312,16 @@
...
  * The issue: If a thermistor falls out or a temperature sensor fails,
  * Marlin can no longer sense the actual temperature. Since a disconnected
  * thermistor reads as a low temperature, the firmware will keep the heater on.
  *
- * If you get "Thermal Runaway" or "Heating failed" errors the
- * details can be tuned in Configuration_adv.h
+ * The solution: Once the temperature reaches the target, start observing.
+ * If the temperature stays too far below the target (hysteresis) for too long,
+ * the firmware will halt as a safety precaution.
  */
```
Comment (Blue-Marlin) [inline Marlin/example_configurations/delta/kossel_xl/Configuration.h:325]: The changes in this comments about Thermal Protection aren't by purpose. Aren't they.

**Comment by Blue-Marlin** [comment]
You are reverting a lot of the small improvements we made in the the last weeks.

**Comment by Grogyan** [comment]
Please don't merge, as the commit removes MAX31855 support.

**Comment by thinkyhead** [comment]
Which settings were changed? If there are just a couple, we can just apply them to the latest config file.

**Comment by thinkyhead** [comment]
@oxivanisher I did a fixup of your commit based on the latest code. Please check out [the updated version](https://github.com/MarlinFirmware/Marlin/pull/3546/files#diff-74332958bdc6d50bdaf9367068e2bb54) and make sure all the settings are correct. I altered some settings to comply with the way Deltas are supposed to generally work, but I may have altered some settings in ways that don't make sense for the Kossel XL.

**Pull request closed without merging**